docs/api/creating-api.md

```diff
@@ -0,0 +1,115 @@
+# Creating the API
+
+## Creating the API on the OneBlink Console
+
+Firstly, we will need to make sure we have something to deploy to when we are ready. Let us go onto the [OneBlink Console](console.oneblink.io). Once we are in our account, go to the "API Hosting" side tab under the Advanced heading. Once here, let us click the plus button in the bottom right hand corner.
```

Comment:
Not sure the link to the url is generated right, Im getting `https://github.com/oneblink/cli/blob/ON-27706/docs/api/console.oneblink.io`

Reply:
And should we link to CP as well?